packages/chips/ChipSet.js

```diff
@@ -41,6 +54,20 @@ export default class ChipSet extends Component {
     };
   }
 
+  handleSelect = (chipId) => {
```

Comment:
Bypassing `foundation.handleChipInteraction` so we don't need to pass the id in `event.details`